src/pages/components/tabs.mdx

```diff
@@ -0,0 +1,103 @@
+---
+title: Tabs
+description: >
+  Tabs organize related content within a single page. This helps users alternate between
+  views without refreshing.
+package: '@zendeskgarden/react-tabs'
+propSheets:
+  - tabs/src/elements/Tabs.tsx
+---
+
+<Usage>
+  <Use>
+    <ul>
+      <li>To provide pre-filtered views of a single dataset</li>
+      <li>To organize related content and controls within a page</li>
+    </ul>
+  </Use>
+  <Misuse>
+    <ul>
+      <li>As a secondary navigation bar that spans multiple pages</li>
+    </ul>
+  </Misuse>
+</Usage>
+
+## Examples
+
+### Default
+
+The basic usage of a Tab component.
+
+import TabsDefault from '../../examples/tabs/TabsDefault.tsx';
+import TabsDefaultCode from '!!raw-loader!../../examples/tabs/TabsDefault.tsx';
+
+<CodeExample code={TabsDefaultCode}>
+  <TabsDefault />
+</CodeExample>
+
+### Vertical
+
+Tabs are horizontal by default but can be manually set to stack vertically.
+
+import TabsVertical from '../../examples/tabs/TabsVertical.tsx';
+import TabsVerticalCode from '!!raw-loader!../../examples/tabs/TabsVertical.tsx';
+
+<CodeExample code={TabsVerticalCode}>
+  <div style={{ width: '320px' }}>
```

Comment:
Why do we have to provide a fixed-width for this example? If it's required lets include it within the `TabsVertical` implementation and use styled-component rather than inline styles.

Reply:
This was sloppy. It was a bandaid to stop this specific component from shifting when Tabs were clicked as the content changed. Good news is putting in more content into the TabPanel pushed the example full width and now the shift doesn't happen anymore. So this is gone.